### PR TITLE
filtered and sorted exam dates

### PR DESCRIPTION
**Pull request by Curryking31**
I filtered the examList to only show the real exams.
Also the exam dates are now sorted chronically.

### CODE REVIEW

File: rogue-thi-app/pages/calendar.js [hunk 1]
```diff
@@ -53,6 +53,17 @@ export async function loadExamList () {
 
       return x
     })
+
+  console.log(tempList)
```
Comment (alexhorn) [inline rogue-thi-app/pages/calendar.js:57]: Can you remove the logging statements? :)

File: rogue-thi-app/pages/calendar.js [hunk 2]
```diff
@@ -53,6 +53,17 @@ export async function loadExamList () {
...
+
+  tempList = tempList.filter((x) => x.modus !== '2')
```
Comment (alexhorn) [inline rogue-thi-app/pages/calendar.js:59]: What exactly does modus 2 mean? Might users be interested in seeing these?

**Review by alexhorn — CHANGES_REQUESTED** [review]

**Comment by Curryking31** [comment]
Added a second commit with comments and without logs 👍

File: rogue-thi-app/pages/calendar.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ export const calendar = rawCalendar.map(x => ({
 
 export async function loadExamList () {
   const examList = await API.getExams()
-  return examList
+  let tempList = examList
```
Comment (M4GNV5) [inline rogue-thi-app/pages/calendar.js:41]: as `filter` returns a new list and `sort` returns the list itself too we do not really need a mutable variable here. A common thing to do is to simply chain the calls as in `return examList.map(...).filter(...).sort(...)`

I guess i can implement that in a new commit. Thanks for the PR :smiley:

**Pull request merged**